nservicebus/azure-servicebus/topologies/index.md

```diff
@@ -0,0 +1,67 @@
+
+Example below demonstrates a publisher called `Publisher` and a subscriber `Subscriber`. `Publisher` can raise two events, `EventA` and `EventB`. A subscriber called `Subscriber` subscribes to both events. Subscriber subscribes by creating two subscription entities, one per event type. Each subscription entity filters out events using a rule associated with the subscription.
+
+This topology has several drawbacks.
+
+1. In order to subscribe for an events, subscribers have to know the publishing endpoint name. As a result of that, causing publisher and subscribers coupling. See Publisher API (link-to-publishers-api).
+1. Multiple subscription entities per subscriber cause polymorphic events to be delivered multiple times to the subscribing endpoint. To handle polymorphic events with `EndpointOrientedTopology` a workaround has to be implemented.
+1. When a single subscriber if offline for an extended period of time and events are not consumed, it can cause events overflow. Since a single topic per publisher is used for all the subscribers, when events overflow is happening, it will affect all subscribers and not just the one that is offline. 
+
+![EndpointOrientedTopology](endpoint-oriented-topology.png "width=50%")
+
+### Forwarding Topology
+
+`ForwardingTopology` is the new topology that was designed to take advantage of several native broker features offered by Azure Service Bus. Unlike `EndpointOrientedTopology`, it doesn't work with a single topic per publisher. All publishers use a topic bundle (link-to-bundles). Subscriptions created under topic bundle are based on subscribers name solely. Each subscription contains multiple rules. A rule per event that subscribing endpoint is interested in. This allows complete decoupling between publishers and subscribers. All messages received by subscription are forwarded to the input queue of the subscriber.
+
+This topology solves the polymorphic events and the events overflow problems `EndpointOrientedTopology` has. 
+
+![ForwardingTopology](forwarding-topology.png "width=50%")
+
+#### Quotas and limitations
+
+With a single Azure Service Bus namespace, `ForwardingTopology` can support up to 2,000 endpoints with up to 2,000 events. Since multiple rules per subscriptions are used, topics cannot be partitioned.
+
+#### Topologies comparison
+
+
+|                                   | EndpointOrientedTopology       | ForwardingTopology |
+|-----------------------------------|--------------------------------|--------------------|
+| Publisher/Subscriber coupling     | yes                            | no                 |
+| Polymorphic events support        | no                             | yes                | 
+| Events overflow protections       | no                             | yes                |
+| Topic partitioning                | yes                            | no                 |
```

Comment:
That's a workaround and not a pleasant one. Topology itself doesn't support it OOTB.